pkg/virt-handler/virtwrap/manager_test.go

```diff
@@ -29,7 +30,12 @@ var _ = Describe("Manager", func() {
 		It("should define and start a new VM", func() {
 			vm := newVM("testvm")
 			mockConn.EXPECT().LookupDomainByName("testvm").Return(mockDomain, libvirt.Error{Code: libvirt.ERR_NO_DOMAIN})
-			xml, err := xml.Marshal(vm.Spec.Domain)
+
+			// we have to make sure that we use correct DomainSpec (from virtwrap)
+			var domainSpec DomainSpec
+			model.Copy(&domainSpec, vm.Spec.Domain)
```

Comment:
Expect(model.Copy(&domainSpec, vm.Spec.Domain)).To(Succeed())

should work here. Since we have to check for errors